common/src/test/java/com/microsoft/identity/common/internal/providers/microsoft/nativeauth/integration/ResetPasswordOAuth2StrategyTest.kt

```diff
@@ -92,6 +92,16 @@ class ResetPasswordOAuth2StrategyTest {
     private val clientId = "079af063-4ea7-4dcd-91ff-2b24f54621ea"
     private val challengeType = "oob redirect"
     private val oobCode = "123456"
+    private val continuationToken = "1234"
```

Comment:
These constants should be in the forrmat ALL_CAPS_WITH_UNDERSCORE. It will make it easier to understand if a variable used in a test is local or constant.

Reply:
Renamed the constants